rpmio/rpmio.c

```diff
@@ -1134,10 +1138,17 @@ static rpmzstd rpmzstdNew(int fdno, const char *fmode)
 	}
 	nb = ZSTD_DStreamInSize();
     } else {					/* compressing */
-	if ((_stream = (void *) ZSTD_createCStream()) == NULL
-	 || ZSTD_isError(ZSTD_initCStream(_stream, level))) {
+	if ((_stream = (void *) ZSTD_createCCtx()) == NULL
+	 || ZSTD_isError(ZSTD_CCtx_setParameter(_stream, ZSTD_c_compressionLevel, level))) {
 	    goto err;
 	}
+
+	if (threads == -1)
+	    threads = rpmExpandNumeric("%{getncpus}");
+	if (threads > 1)
```

Comment:
After all I think

```suggestion
	if (threads > 0)
```

should be used.